### DISCUSSION TITLE
Inconsistent repository primary branch name

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#127] [Ideas]
The 11 repos under the `space-ros` organization use 3 different names for their primary primary branches:
- 7 use `main`
- 3 use `master` (`.github`, `ikos`, `spaceros.org`)
- 1 uses `rolling` (looking at you, `docs`)

It isn't a requirement that they all match, and in the case of forks like `ikos` shouldn't be changed, but we should probably at least update `docs`, right? What about `.github` and `spaceros.og`? I believe those are the repos with restricted permissions (see #120), that might require escalation to the foundation.

**Comment by ivanperez-keera** [comment]
The ikos fork has to go. There's no point in having a fork of ikos within Space ROS. We can just pin the specific version from ikos' repo: https://github.com/NASA-SW-VnV/ikos.

**Reply by Bckempa** [reply]
Ok, does https://github.com/space-ros/docker/issues/99 need to be resolved before we delete the fork?

**Reply by ivanperez-keera** [reply]
No.

**Reply by Bckempa** [reply]
Ok, well it isn't referenced by anything so I think we're good to nuke it at any time. Do you have permission or do we need @EzraBrooks ?

**Reply by ivanperez-keera** [reply]
I don't have permissions. @EzraBrooks, can you please remove the ikos repo from the space-ros organization?

**Reply by EzraBrooks** [reply]
I do not have permissions either.

**Comment by Bckempa** [comment]
Following @ivanperez-keera's lead for what should be an issue vs a discussion, I've moved this here. We can split off issues as we disposition each repository's branch structure.
